robots/tank.h

```diff
@@ -47,13 +64,48 @@ class Tank
                 });
     }
 
+    void controlWithThumbsticks(HID::Joystick &joystick)
```

Comment:
Hah now that's truly a psychotic control scheme

Reply:
It's not particularly practical ;-). I ended up sticking it in because it's handy for debugging, because you can control each wheel individually.

Reply:
:+1: